systems/robotInterfaces/QPLocomotionPlan.m

```diff
@@ -338,111 +349,11 @@
 
       inds = obj.plan_shift_body_motion_inds;
       for j = 1:length(qp_input.body_motion_data)
-        qp_input.body_motion_data(j).coefs(inds,:,end) = qp_input.body_motion_data(j).coefs(inds,:,end) - obj.plan_shift(inds);
+        qp_input.body_motion_data(j).coefs(inds,:,end) = qp_input.body_motion_data(j).coefs(inds,:,end) - repmat(obj.plan_shift(inds), [1, size(qp_input.body_motion_data(j).coefs, 2)]);
```

Comment:
we had to add the repmat because coefs is no longer size 1 along the second dimension